app/components/becoming_a_teacher_review_component.html.erb

```diff
@@ -1,9 +1,5 @@
 <% if show_missing_banner? %>
-  <%= render(SectionMissingBannerComponent, section: :becoming_a_teacher, text: t('review_application.becoming_a_teacher.incomplete')) do %>
-    <%= link_to candidate_interface_becoming_a_teacher_edit_path, class: 'govuk-link' do %>
-      <%= t('review_application.becoming_a_teacher.link_html') %>
-    <% end %>
-  <% end %>
+  <%= render(SectionMissingBannerComponent, section: :becoming_a_teacher, section_path: candidate_interface_becoming_a_teacher_edit_path, text: t('review_application.becoming_a_teacher.incomplete')) %>
```

Comment:
This + all of the other ones: do we still need to pass in `text: t('review_application.becoming_a_teacher.incomplete')` if we're passing in `section` above?

Reply:
Oooooh!